modules/nw-multus-create-master-interface-bridge-cni.adoc

```diff
@@ -0,0 +1,213 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-additional-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-multus-create-master-interface-bridge-cni_{context}"]
+= Creating a subinterface based on a bridge master interface in a container namespace
+
+Creating a subinterface can be applied to other types of interfaces. Follow this procedure to create a subinterface based on a bridge master interface in a container namespace.
+
+.Prerequisites
+* You have installed the {product-title} CLI (oc).
+* You are logged in to the {product-title} cluster as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a dedicated container namespace where you want to deploy your pod by running the following command:
++
+[source,terminal]
+----
+$ oc new-project test-namespace
+----
+
+. Using the following YAML example, create a bridge `NetworkAttachmentDefinition` custom resource (CR) file named `bridge-nad.yaml`:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: bridge-network
+spec:
+  config: '{
+    "cniVersion": "0.4.0",
+    "name": "bridge-network",
+    "type": "bridge",
+    "bridge": "br-001",
+    "isGateway": true,
+    "ipMasq": true,
+    "hairpinMode": true,
+    "ipam": {
+      "type": "host-local",
+      "subnet": "10.0.0.0/24",
+      "routes": [{"dst": "0.0.0.0/0"}]
+    }
+  }'
+----
+
+. Run the following command to apply the `NetworkAttachmentDefinition` CR to your OpenShift cluster:
++
+[source,terminal]
+----
+$ oc apply -f bridge-nad.yaml
+----
+
+.  Verify that the `NetworkAttachmentDefinition` CR has been created successfully by running the following command:
++
+[source,terminal]
+----
+$ oc get network-attachment-definitions
+----
++
+.Expected output
```

Comment:
We use "Example output" consistently
```suggestion
.Example output
```

Reply:
I thought Expected output would apply in this case as this is a worked example end-to-end and this is the expected output based on this worked example. It is not example output as such.